modules/gdscript/gdscript_tokenizer_buffer.cpp

```diff
@@ -296,7 +296,7 @@ Vector<uint8_t> GDScriptTokenizerBuffer::parse_code_string(const String &p_code,
 	encode_uint32(identifier_map.size(), &contents.write[0]);
 	encode_uint32(constant_map.size(), &contents.write[4]);
 	encode_uint32(token_lines.size(), &contents.write[8]);
-	encode_uint32(0, &contents.write[12]); // Unused, kept for compatibility. Please remove at next `TOKENIZER_VERSION` increment.
+	encode_uint32(0, &contents.write[12]);
```

Comment:
You just removed the comment. You should remove the whole line instead.

Reply:
Ok, I have done this.

Reply:
To be clear, do you want me to change the size of the header or leave the header the same size?

Reply:
Hmm not sure, the code reading this probably needs adjustment as well. Someone who works on GDScript should be able to give better advice.

Reply:
Yes, you should resize the header and recalculate the offsets, in both encoding and decoding code. You should **not** replace `resize()` with `resize_zeroed()`, as it is less performant.